test/verify/check-kdump

```diff
@@ -103,25 +109,57 @@ class TestKdump(MachineCase):
         self.enableKdump()
         self.rebootMachine()
         self.browser.wait_visible("#app")
-
-        # we should have the amount of memory reserved that we indicated
-        self.browser.wait_in_text("#app", "256 MiB")
-        # service should start up properly and the button should be on
-        self.browser.wait_in_text("#app", "Service is running")
-        self.browser.wait_in_text(onOffSelectorActive, "On")
+        self.enableLocalSsh()
+
+        if self.machine.image not in ["rhel-7-6-distropkg"]:
+            # generate a valid kdump config with ssh target
+            settingsLink = "a:contains('locally in /var/crash')"
+            self.browser.wait_present(settingsLink)
+            self.browser.wait_visible(settingsLink)
```

Comment:
As this is React, where we usually don't show/hide, but render or not, does this really need to be checked separately?